src/lint/lint.ts

```diff
@@ -84,6 +84,6 @@ export async function lint(options: Options, config: ProjectConfig) {
       message += ` ${infos.length} ${chalk.green('info')} messages`;
     }
     console.log(`\n\nFound ${message}.`);
-    return new CommandResult(1);
+    return new CommandResult(errors.length > 0 ? 1 : 0);
```

Comment:
Add a comment to explain this a bit more. Or, better, yet, move it into a descriptive variable name:

```js
const shouldCommandFail = errors.length > 0;
return new CommandResult(shouldLintFail ? 1 : 0);
```